app/presenters/tslr_claim_csv_row.rb

```diff
@@ -0,0 +1,29 @@
+require "delegate"
+
+class TslrClaimCSVRow < SimpleDelegator
```

Comment:
Can we be consistent with `CSV` vs `Csv` in class names, please?

Reply:
Ah, good call. I made the decision to go with `Csv`